contracts/network/BancorNetwork.sol

```diff
@@ -510,7 +520,7 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
         // add the pool collection to the reverse pool collection lookup
         _collectionByPool[token] = poolCollection;
 
-        emit PoolAdded({ pool: token, poolCollection: poolCollection });
+        emit PoolCreated({ pool: token, poolCollection: poolCollection });
```

Comment:
PoolAdded should be triggered here as well

Reply:
👍